src/main/java/org/bigredbands/mb/controllers/PDFGenerator.java

```diff
@@ -240,154 +210,92 @@ public void createPDF(DrillInfo drillInfo, File file) throws IOException {
                     yPosition -= lineSpacing * commentFontSize;
                 }
 
+                // Divide per-rank commands into columns
+                int numColumns = 3;
+                float columnWidth = (pageWidth - 2 * textMarginX) / (float) numColumns;
+                float commandMaxWidth = columnWidth * 0.95f;
+
+                // Get the ranks grouped by unique move
+                List<ConsolidatedDrillInstruction> rankGroups = move.getCommands().entrySet()
+                    // First, convert to map of <list of ranks> -> <list of commands>
+                    .stream().collect(
+                        Collectors.groupingBy(
+                            // TODO: we should group by the string generated by
+                            // the list of commands, NOT the values of the
+                            // CommandPairs. Two direct-to-point CommandPairs
+                            // with different destinations aren't considered
+                            // "equal", but will render identically in PDF
```

Comment:
Does it make sense to have two ranks that have the same direct-to-point moves that start/end at different places receive the same instructions?  Or since they're going to different places should it remain different?

Reply:
Both of them will show up as "DTP `<count>`" in the instruction text, so I'd probably want to consolidate them there. Then again, we might want to consider adding destination information for direct to point moves?

I'll leave the TODO in the code for now, we can always revisit that question if/when someone adds a "fix".